app/views/layouts/smart_answers.html.erb

```diff
@@ -14,6 +14,13 @@
 
     <%= render 'smartanswer_metadata' %>
   </main>
+
+  <% if @presenter.finished? %>
```

Comment:
What's `finished`? It doesn't seem to be defined - it's the reason this build is failing.

Reply:
it was meant to be [this code here](https://github.com/alphagov/smart-answers/blob/ad76605d8c5ba8c9bd6238f51befddfa6a72b407/app/presenters/flow_presenter.rb#L27),  but it doesn't seem like it works outside of the smart answers. I will investigate a better solution.